src/test/ui/generic-associated-types/impl_bounds.stderr

```diff
@@ -16,28 +16,8 @@ LL |     type B<'a, 'b> where 'a: 'b;
 LL |     type B<'a, 'b> = (&'a(), &'b ()) where 'b: 'a;
    |                                                ^^ impl has extra requirement `'b: 'a`
 
-error[E0478]: lifetime bound not satisfied
```

Comment:
This error is never emitted since we return `ErrorGuaranteed` in `compare_type_predicate_entailment`, which means we never get to `check_type_bounds` which is where this error is reported.